db/migrate/20190925065737_add_email_field_to_solidus_gdpr_request.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class AddEmailFieldToSolidusGdprRequest < ActiveRecord::Migration[5.1]
+  def change
+    add_column :spree_gdpr_requests, :email, :string
+
+    reversible do |dir|
+      dir.up do
+        Spree::GdprRequest.all.each do |gdpr_request|
+          gdpr_request.update!(email: gdpr_request.user.email)
```

Comment:
You have removed the `user` association from `Spree::GdprRequest`, so `gdpr_request.user` is not going to work anymore. You can use `Spree::User.find(gdpr_request.user_id).email` instead.

Reply:
I think you should use `update_columns` here: it's much faster because it skips all the AR logic, which you don't need in this case.